pkg/cmd/issue/close/close_test.go

```diff
@@ -2,188 +2,280 @@ package close
 
```

Comment:
Migrated these tests from old style to new style hence the large number of line changes.